cmd/ethrex/ethrex.rs

```diff
@@ -32,6 +30,8 @@ mod decode;
 mod networks;
 
 const DEFAULT_DATADIR: &str = "ethrex";
+const DEFAULT_KEYDIR: &str = "ethrex_key";
```

Comment:
Don't we want to add this to the data dir passed via cli? `--datadir`

Reply:
Done.